srcCpp/perftest_publisher.cxx

```diff
@@ -34,7 +34,7 @@ bool perftest_cpp::_testCompleted_scan = true; // In order to enter into the sca
 const char *perftest_cpp::_LatencyTopicName = "Latency";
 const char *perftest_cpp::_AnnouncementTopicName = "Announcement";
 const char *perftest_cpp::_ThroughputTopicName = "Throughput";
-const DDS_Duration_t timeout_wait_for_ack = {0, 100000000};
+const int timeout_wait_for_ack_nsec = 100000000;
```

Comment:
timeout_wait_for_ack_nsec should be unsigned long
timeout_wait_for_ack_sec should be long

Reply:
Done